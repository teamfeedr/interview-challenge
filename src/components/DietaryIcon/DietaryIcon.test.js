import React from 'react';
import {shallow} from 'enzyme';

import DieataryIcon from './DietaryIcon';

describe('the DietaryIcon component', () => {
    it('renders with a diet', () => {
        const dietaryIcon = shallow(<DieataryIcon diet={'ti'}/>);
        expect(dietaryIcon).toMatchSnapshot();
    })
})