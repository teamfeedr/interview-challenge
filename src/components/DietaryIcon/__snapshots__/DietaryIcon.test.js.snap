// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the DietaryIcon component renders with a diet 1`] = `
<span
  className="dietary"
>
  ti
</span>
`;
